modmesh/system.py

```diff
@@ -83,6 +82,19 @@ def _run_viewer(argv):
     wm.windowTitle = "Modmesh Viewer"
     wm.resize(w=1000, h=600)
     wm.show()
+
+    config = {
+        'appEnvName': 'main',
+        'redirectStdOutFile': 'stdout_viewer.txt',
```

Comment:
Why do we have file names?  stdout and stderr from Python should be redirected to the Qt widget but not any files on the disks.  We do not want to pay for the additional cost of the storage.